File: ProjectFeedget/app/src/main/java/kr/mashup/feedget/ui/register/register_content/Contract.java
package kr.mashup.feedget.ui.register.register_content;

import android.view.inputmethod.InputMethodManager;

import kr.mashup.feedget.databinding.ActivityRegisterBinding;

interface Contract {

    interface View {

        void initViews();
    }

    interface Presenter {
        void init();
    }

}

File: ProjectFeedget/presentation/src/main/java/kr/mashup/feedget/presentation/Contract.kt
package kr.mashup.feedget.presentation

interface RootView{

}
interface RootPresenter{

}

interface View<out T : RootPresenter> : RootView{
    val presenter : T
}

interface Presenter<out T : RootView>: RootPresenter{
    val view : T
}
